src/components/evaluation/UpdateEvaluations.vue

```diff
@@ -432,6 +459,13 @@ export default {
     },
     selectInstructor(suggestion) {
       this.selectedInstructor = suggestion
+      if (this.selectedInstructor) {
+        console.log(this.selectedInstructor)
```

Comment:
@pauline2k - you want this `console.log` here?

Reply:
I do not! one sec.